src/util.js

```diff
@@ -430,7 +430,7 @@ const util = {
       return os.cpus().length;
     }
 
-    return navigator.hardwareConcurrency || 1;
+    return (typeof navigator !== 'undefined' && navigator.hardwareConcurrency) || 1;
```

Comment:
I appreciate you trying to fix this problem, but just so you know: it doesn't fix the Cloudfare issue because it's still accessing a prohibited API. The mere attempt to read `navigator` is what's not allowed.
I tried to explain that in my pull request.
I will try to check with cloudflare if they could allow code like this to "pass"... but I thought you would be interested in knowing that.

Reply:
I understand - but even after your PR, you still had to shim `util.isEdge`, afaiu - after this PR and #1474, you can shim `util.getHardwareConcurrency` instead :)

Other than that, please do check with Cloudflare, as this is the standard way to check for the existence of global objects, it shouldn't cause issues.